src/systemTest/java/com/flightstats/hub/system/functional/WebhookErrorTest.java

```diff
@@ -0,0 +1,156 @@
+package com.flightstats.hub.system.functional;
+
+import com.flightstats.hub.kubernetes.HubLifecycle;
+import com.flightstats.hub.model.Webhook;
+import javax.inject.Inject;
+
+import com.flightstats.hub.model.WebhookCallbackSetting;
+import com.flightstats.hub.system.ModelBuilder;
+import com.flightstats.hub.system.config.DependencyInjector;
+import com.flightstats.hub.system.service.CallbackService;
+import com.flightstats.hub.system.service.ChannelService;
+import com.flightstats.hub.system.service.WebhookService;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.RepeatedTest;
+
+import java.util.Comparator;
+
+import static com.flightstats.hub.util.StringUtils.randomAlphaNumeric;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@Slf4j
+//@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Did this work? Compiler did not complain that @BeforeAll method is non static?

Reply:
The tests pass, so I guess it's unnecessary. I'll delete rather than comment it out.

Reply:
I'll double check to make sure the repeated tests are behaving like I expect them to behave.

Reply:
RE: static @BeforeAll, from the docs: ```Such methods are inherited (unless they are hidden or overridden) and must be static (unless the "per-class" test instance lifecycle is used).```

Reply:
Working on fixing this now. I broke the functionality in other ways, too. :P